sklearn/feature_selection/rfe.py

```diff
@@ -66,8 +69,8 @@ class RFE(BaseEstimator, MetaEstimatorMixin, SelectorMixin):
         The mask of selected features.
 
     ranking_ : array of shape [n_features]
-        The feature ranking, such that `ranking_[i]` corresponds to the \
-        ranking position of the i-th feature. Selected (i.e., estimated \
+        The feature ranking, such that `ranking_[i]` corresponds to the
```

Comment:
I realize it was like this before, but double ticks are probably the formatting we want to go for.